### PR TITLE
Show cookie expiry if cookieStore available.

### PR DESCRIPTION
**Pull request by kdzwinel**
expiry

Limitations:
- only JS set cookies (on header set cookies we have httponly flag)
- only Chromium (CookieStore API)

### CODE REVIEW

File: privacy-protections/storage-blocking/3rdparty.js [hunk 1]
```diff
@@ -7,14 +7,22 @@
...
     commonTests.push({
         id: `JS cookie (3rd party ${trackingDomain ? 'tracking' : 'safe'} script)`,
         store: (data) => {
-            document.cookie = `tp${trackingDomain ? 't' : 's'}data=${data}; expires= Wed, 21 Aug 2030 20:00:00 UTC;`;
+            document.cookie = `${cookieName}=${data}; expires= Wed, 21 Aug 2030 20:00:00 UTC;`;
         },
         retrive: () => {
             return trackingDomain ? document.cookie.match(/tptdata=([0-9]+)/)[1] : document.cookie.match(/tpsdata=([0-9]+)/)[1];
+        },
+        extra: () => {
```
Comment (kdzwinel) [inline privacy-protections/storage-blocking/3rdparty.js:20]: I introduced an "extra" callback that can show additional info next to the result. This info doesn't go into the results object.

File: privacy-protections/storage-blocking/iframe.js [hunk 2]
```diff
@@ -31,33 +31,28 @@ function storeData (randomNumber) {
 }
 
 function retrieveData () {
-    return Promise.all(commonTests.map(test => {
+    return Promise.all(commonTests.map(async (test) => {
         try {
             const result = test.retrive();
+            const value = (result instanceof Promise) ? await result : result;
```
Comment (sammacbeth) [inline privacy-protections/storage-blocking/iframe.js:37]: Await also works on non-promises.
```suggestion
            const value = await result;
```

File: privacy-protections/storage-blocking/iframe.js [hunk 3]
```diff
@@ -31,33 +31,28 @@ function storeData (randomNumber) {
...
-                        error: e.message
-                    }));
-            } else {
-                return Promise.resolve({
-                    test: test.id,
-                    value: result
-                });
+            if (test.extra) {
+                const extraResult = test.extra();
+                extra = (extraResult instanceof Promise) ? await extraResult : extraResult;
```
Comment (sammacbeth) [inline privacy-protections/storage-blocking/iframe.js:42]:
```suggestion
                extra = await extraResult;
```
As above

**Review by sammacbeth — APPROVED** [review]
Looks good. One minor comment, but non-blocking.

File: privacy-protections/storage-blocking/iframe.js (hunk 2 above)
Comment (kdzwinel) [reply privacy-protections/storage-blocking/iframe.js:37]: TIL, this is great. Thanks!

**Pull request merged**